src/pages/EnablePayments/TermsStep.js

```diff
@@ -17,20 +17,23 @@ import Text from '../../components/Text';
 import ShortTermsForm from './TermsPage/ShortTermsForm';
 import LongTermsForm from './TermsPage/LongTermsForm';
 import FormAlertWithSubmitButton from '../../components/FormAlertWithSubmitButton';
+import walletTermsPropTypes from './walletTermsPropTypes';
+import userWalletPropTypes from './userWalletPropTypes';
 
 const propTypes = {
     /** Comes from Onyx. Information about the terms for the wallet */
-    walletTerms: PropTypes.shape({
-        /** Any additional error message to show */
-        errors: PropTypes.objectOf(PropTypes.string),
-    }),
+    // eslint-disable-next-line react/no-unused-prop-types
+    walletTerms: PropTypes.objectOf(walletTermsPropTypes),
```

Comment:
NAB since I'm sure there's good reason for this but why are we including this prop if it's not used in this component?

Reply:
Looks like its used here: https://github.com/Expensify/App/blob/6c8328bdc872b9579f38b67180eaff16e2c99106/src/pages/EnablePayments/TermsStep.js#L71

But the linter can't understand that.

Reply:
ahh my bad, thanks for clarifying!